commerce_coordinator/apps/commercetools/catalog_info/edx_utils.py

```diff
@@ -40,6 +40,16 @@ def is_edx_lms_order(order: CTOrder) -> bool:
     return len(get_edx_items(order)) >= 1
 
 
+def get_course_mode_from_ct_order(order: CTOrder) -> str:
+    mode = 'verified'
+    for line_item in order.line_items:
+        for attribute in line_item.variant.attributes:
+            if attribute.name == 'mode':
+                mode = attribute.value
+
+    return mode
```

Comment:
We do not need to loop over all of the line items here. Ideally, this function should take a line item as an argument and only retrieve its `mode` attribute value with a fallback of `verified` as value.

Reply:
Addressed